packages/app/src/cli/services/dev/migrate-app-module.test.ts

```diff
@@ -17,30 +16,55 @@ function getLocalExtension(attributes: Partial<LocalSource> = {}) {
 function getRemoteExtension(attributes: Partial<RemoteSource> = {}) {
   return {
     uuid: '1234',
-    type: 'payments_app_credit_card',
     title: 'a-different-extension',
     ...attributes,
   } as unknown as RemoteSource
 }
+const defaultMap = {
+  payments_extension: [
+    'payments_app',
+    'payments_app_credit_card',
+    'payments_app_custom_credit_card',
+    'payments_app_custom_onsite',
+    'payments_app_redeemable',
+  ],
```

Comment:
should we include the other extension types we've added? ie. flow and ui extensions

Reply:
The function we are testing here is generic, so by testing just a few extension types we cover all cases.